src/Raven.Client/Documents/Subscriptions/SubscriptionWorker.cs

```diff
@@ -586,6 +611,10 @@ private async Task RunSubscriptionAsync()
                 }
                 catch (Exception ex)
                 {
+                    while (_recentExceptions.Count > 10)
```

Comment:
If you want to keep 10 messages then it should be `>=` because you are adding new message 2 lines below

Reply:
ok